[2025-02-18T04:31:00.966+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:31:00.986+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-14T00:00:00+00:00 [queued]>
[2025-02-18T04:31:00.998+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-14T00:00:00+00:00 [queued]>
[2025-02-18T04:31:00.999+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:31:01.009+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-14 00:00:00+00:00
[2025-02-18T04:31:01.017+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=317) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:31:01.018+0000] {standard_task_runner.py:72} INFO - Started process 322 to run task
[2025-02-18T04:31:01.018+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-14T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmp6uh5qrc3']
[2025-02-18T04:31:01.022+0000] {standard_task_runner.py:105} INFO - Job 28: Subtask hello_task
[2025-02-18T04:31:01.034+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:01.059+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-14T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:31:01.060+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:31:01.102+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-14T00:00:00+00:00'
[2025-02-18T04:31:01.103+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:31:01.110+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:31:01.110+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:31:01.115+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:31:01.115+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-14T00:00:00+00:00, execution_date=20240314T000000, start_date=20250218T043100, end_date=20250218T043101
[2025-02-18T04:31:01.158+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:31:01.167+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T04:31:01.167+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-18T16:34:04.155+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T16:34:04.175+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-14T00:00:00+00:00 [queued]>
[2025-02-18T16:34:04.182+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-03-14T00:00:00+00:00 [queued]>
[2025-02-18T16:34:04.184+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T16:34:04.196+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-03-14 00:00:00+00:00
[2025-02-18T16:34:04.201+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1601) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T16:34:04.202+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-03-14T00:00:00+00:00', '--job-id', '396', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpy45vg8xx']
[2025-02-18T16:34:04.204+0000] {standard_task_runner.py:72} INFO - Started process 1606 to run task
[2025-02-18T16:34:04.204+0000] {standard_task_runner.py:105} INFO - Job 396: Subtask hello_task
[2025-02-18T16:34:04.224+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:04.248+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-03-14T00:00:00+00:00 [running]> on host 8720d9c228eb
[2025-02-18T16:34:04.249+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T16:34:04.319+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-03-14T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-14T00:00:00+00:00'
[2025-02-18T16:34:04.320+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T16:34:04.349+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T16:34:04.349+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T16:34:04.360+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T16:34:04.361+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-03-14T00:00:00+00:00, execution_date=20240314T000000, start_date=20250218T163404, end_date=20250218T163404
[2025-02-18T16:34:04.432+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T16:34:04.451+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T16:34:04.452+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
